### PR TITLE
Docs – Deployment to Firebase Cloud Functions

### PR DESCRIPTION
**Pull request by lukepighetti**
This pull request provides examples for how to run Apollo Server on Firebase Cloud Functions.

Cheers,
Luke Pighetti

### CODE REVIEW

**Comment by lukepighetti** [comment]
I signed it!

File: docs/source/deployment/firebase.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+```
+firebase init
+```
+
+
+Install required dependencies.
+
+```
+cd functions
+npm install apollo-express graphql express
```
Comment (jthegedus) [inline docs/source/deployment/firebase.md]: `apollo-server-express`

**Comment by piuccio** [comment]
I published [an article](https://medium.com/@piuccio/running-apollo-server-on-firebase-cloud-functions-265849e9f5b8) describing my setup for deployment to firebase cloud functions, it might be of interests if you're planning to update the PR

**Comment by glasser** [comment]
It would be great to have examples for deployment to Firebase Cloud Functions, but I'm sure this will need to be updated to work with Apollo Server 4 (currently a Release Candidate, to be released as v4.0.0 very soon).

**Pull request closed without merging**